tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/sr_evaluator.py

```diff
@@ -0,0 +1,572 @@
+                blob_list = list(model.glob('*{}.blob'.format(self.default_model_suffix)))
+                if not model_list and not blob_list:
+                    model_list = list(model.glob('*.xml'))
+                    blob_list = list(model.glob('*.blob'))
+                    if not model_list:
+                        model_list = blob_list
+            if not model_list:
+                raise ConfigError('Suitable model for {} not found'.format(self.default_model_suffix))
+            if len(model_list) > 1:
+                raise ConfigError('Several suitable models for {} found'.format(self.default_model_suffix))
+            model = model_list[0]
+            print_info('{} - Found model: {}'.format(self.default_model_suffix, model))
+        if model.suffix == '.blob':
+            return model, None
+        weights = get_path(network_info.get('weights', model.parent / model.name.replace('xml', 'bin')))
+        print_info('{} - Found weights: {}'.format(self.default_model_suffix, weights))
+        return model, weights
+
+    def load_network(self, network, launcher):
+        self.network = network
+        self.exec_network = launcher.ie_core.load_network(network, launcher.device)
+
+    def set_input_and_output(self):
+        has_info = hasattr(self.exec_network, 'input_info')
+        input_info = self.exec_network.input_info if has_info else self.exec_network.inputs
+        input_blob = next(iter(input_info))
+        with_prefix = input_blob.startswith(self.default_model_suffix)
+        if self.input_blob is None or with_prefix != self.with_prefix:
+            if self.input_blob is None:
+                output_blob = next(iter(self.exec_network.outputs))
+            else:
+                output_blob = (
+                    '_'.join([self.default_model_suffix, self.output_blob])
+                    if with_prefix else self.output_blob.split(self.default_model_suffix + '_')[-1]
+                )
+            self.input_blob = input_blob
+            self.output_blob = output_blob
+            self.with_prefix = with_prefix
+
+    def load_model(self, network_info, launcher, log=False):
+        if 'onnx_model' in network_info:
+            network_info.update(launcher.config)
+            model, weights = launcher.convert_model(network_info)
+        else:
+            model, weights = self.automatic_model_search(network_info)
+        if weights is not None:
+            self.network = launcher.read_network(str(model), str(weights))
+            self.exec_network = launcher.ie_core.load_network(self.network, launcher.device)
+        else:
+            self.exec_network = launcher.ie_core.import_network(str(model))
+        self.set_input_and_output()
+        if log:
+            self.print_input_output_info()
+
+
+def create_model(model_config, launcher, delayed_model_loading=False):
+    launcher_model_mapping = {
+        'dlsdk': ModelDLSDKModel,
+        'tf': ModelTFModel,
+        'dummy': DummyModel
```

Comment:
why do you need dummy model if you have only one part?
Usually this trick used for multi stage pipelines for debugging purposes (isolate influence of first part on others)

Reply:
fixed